### ISSUE TITLE
accessibility: GridView of files and folders is not accessible for screenreaders

### ISSUE THREAD

**Issue by beqabeqa473** [files-community/Files#4757]
Hello.

A GridView for displaying files and folders is not accessible for screenreaders.
columns and rows are not readable.

to reproduce an issue, you can launch narrator by pressing ctrl+shift+enter, open filels application, open some folder in it and try to navigate in a Grid by arrow keys.

expected result:
screenreader should read names of files and folders and all other possible columns in a row.

Actual result:
Screenreader is silent there.

Final words:
Files application is very good replacement for windows explorer, but because of a gap of accessibility in such areas, blind people cannot use this application.

**Comment by yaichenbaum** [comment]
@beqabeqa473 What version of Files are you using? I just tested with the latest version of Files, and the screen reader is working fine in the grid view. By any chance, are you using the details layout?

**Comment by beqabeqa473** [comment]
@yair100 I downloaded 1.5 version from github.
Yes, Details view was selected by default.

**Comment by yaichenbaum** [comment]
I am sorry about the inconvenience especially since accessibility is priority number one for us. I am happy to report that this issue is already resolved in `main` and will be available when we ship the next update, in the meantime, the grid view layout should be accessible with the screen reader.

**Comment by beqabeqa473** [comment]
I am ready to test and report issues if they still persist, because accessibility is high priority for me also, as i am blind person.

i found some other issues which could be already resolved.

do you have some beta channel or nightly builds so i could download and test the latest version?

thanks for prioritising accessibility.

**Comment by yaichenbaum** [comment]
@beqabeqa473 Please make sure to report other issues that you find, it's likely that many of them are already resolved in the dev builds, but it's better to report them in case we missed them. You can also join our Discord server to get access to preview builds.
